docs/source/conf.py

```diff
@@ -59,7 +60,7 @@ def setup(app):
 
 # General information about the project.
 project = 'Site Reliability Guide for mybinder.org'
-copyright = '2017, Binder Team'
+copyright = '%d, Binder Team' % datetime.now().year
```

Comment:
```suggestion
copyright = '2017 - %d, Binder Team' % datetime.now().year
```

IANAL. This way we preserve the starting date which is nice

Reply:
I tried committing this but it wouldn't let me :-/